internal/cmd/mongodbflex/backup/describe/describe.go

```diff
@@ -0,0 +1,141 @@
+		Long:  "Shows details of a backup for a MongoDB Flex instance.",
+		Example: examples.Build(
+			examples.NewExample(
+				`Get details of a backup with ID "xxx" for a MongoDB Flex instance with ID "yyy"`,
+				"$ stackit mongodbflex backup describe xxx --instance-id yyy"),
+			examples.NewExample(
+				`Get details of a backup with ID "xxx" for a MongoDB Flex instance with ID "yyy" in JSON format`,
+				"$ stackit mongodbflex backup describe xxx --instance-id yyy --output-format json"),
+		),
+		Args: args.SingleArg(backupIdArg, nil),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd, args)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+
+			if err != nil {
+				return fmt.Errorf("describe backup for MongoDB Flex instance: %w", err)
+			}
+
+			return outputResult(p, cmd, model.OutputFormat, *resp.Item)
+		},
+	}
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().Var(flags.UUIDFlag(), instanceIdFlag, "Instance ID")
+
+	err := flags.MarkFlagsRequired(cmd, instanceIdFlag)
+	cobra.CheckErr(err)
+}
+
+func parseInput(p *print.Printer, cmd *cobra.Command, inputArgs []string) (*inputModel, error) {
+	backupId := inputArgs[0]
+
+	globalFlags := globalflags.Parse(p, cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	model := inputModel{
+		GlobalFlagModel: globalFlags,
+		InstanceId:      flags.FlagToStringValue(p, cmd, instanceIdFlag),
+		BackupId:        backupId,
+	}
+
+	if p.IsVerbosityDebug() {
+		modelStr, err := print.BuildDebugStrFromInputModel(model)
+		if err != nil {
+			p.Debug(print.ErrorLevel, "convert model to string for debugging: %v", err)
+		} else {
+			p.Debug(print.DebugLevel, "parsed input values: %s", modelStr)
+		}
+	}
+
+	return &model, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *mongodbflex.APIClient) mongodbflex.ApiGetBackupRequest {
+	req := apiClient.GetBackup(ctx, model.ProjectId, model.InstanceId, model.BackupId)
+	return req
+}
+
+func outputResult(p *print.Printer, cmd *cobra.Command, outputFormat string, backup mongodbflex.Backup) error {
+	switch outputFormat {
+	case print.JSONOutputFormat:
+		details, err := json.MarshalIndent(backup, "", "  ")
+		if err != nil {
+			return fmt.Errorf("marshal backup for MongoDB Flex backup: %w", err)
+		}
+		cmd.Println(string(details))
+
+		return nil
+	default:
+		table := tables.NewTable()
+		table.AddRow("ID", *backup.Id)
+		table.AddSeparator()
+		table.AddRow("CREATED AT", *backup.StartTime)
+		table.AddSeparator()
+		table.AddRow("EXPIRES AT", *backup.EndTime)
+		table.AddSeparator()
+		table.AddRow("BACKUP SIZE", bytesize.New(float64(*backup.Size)))
+
```

Comment:
As discussed offline, this could include the restore status

Reply:
done